src/get-started/light-client.md

```diff
@@ -356,50 +474,71 @@ Otherwise, transactions cannot be verified against the proof:
 if (data.wallet_history.transactions.length !==
     transactionsMetaData.length)
 {
-  throw new Error('Transactions can not be verified')
+  throw new Error('Transactions cannot be verified')
 }
 ```
 
 Next, we validate each transaction. For this purpose we iterate them in the
-array and first check them according to their structure. This check allows us
+array and check their structure in several steps. This check allows us
 to confirm that a transaction of a certain type is present at a definite place
-in the array. In our example, for the sake of brevity, we provide structure
-definition of an issue-type transaction only. However, note that all the
-transaction types of the service should be defined.
+in the array.
+
+In our example, for the sake of brevity, we provide structure
+definition of only one transaction type. This type is a transaction of token
+issuance from our [Cryptocurrency Advanced Service][cryptocurrency-advanced].
+However, note that to perform a real check, all the transaction types of the
+service should be defined.
+
+The check of the transaction structure comprises the following steps:
 
-Next, we validate each transaction signature with `Transaction.verifySignature`
-method.
+- calculate a hash from the transaction buffer with the `Exonum.hash`
```

Comment:
Nit: Shouldn't sentences in this list be capitalized?

Reply:
I do not think capitalization is necessary here. The steps we enlist are in fact a contuniation of the introductory part above them. Do we have any approved rules concerning this issue? If not, I'll remember to reflect it in our style guide in future.